components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -925,14 +925,15 @@ def wrapObjectTemplate(templateBody, nullValue, isDefinitelyObject, type,
 
         if failureCode is None:
             unwrapFailureCode = '''throw_type_error(*cx, "This object is not \
-                    an instance of ReadableStream.");\n'''
+                    an instance of ReadableStream.");\nreturn false;\n'''
```

Comment:
Is this codepath even happening, given that we pass `use_builtin_readable_stream=False`?